src/components/select/_select.scss

```diff
@@ -141,17 +148,18 @@
         @include focus-outline('border');
       }
 
+      ~ .#{$prefix}--select__arrow {
+        bottom: auto;
+        top: 1rem;
+      }
+
       &[data-invalid] {
-        outline: 1px solid $support-01;
```

Comment:
Was this moved to only show on focus on purpose? I believe it should stay.

Reply:
I originally wrote it to be outlined all the time if the data was invalid but reverted it after [it was pointed out in Slack](https://ibm-research.slack.com/archives/C2K6RFJ1G/p1529687021000550?thread_ts=1529595199.000502&cid=C2K6RFJ1G). I don't think we came to a definitive conclusion about it though

Reply:
Is there a reason its not following the red underline of the other input components?

Reply:
Just following the existing styles for the valid inline input fields:

valid inline input

![image](https://user-images.githubusercontent.com/8265238/41927892-2349eae8-7939-11e8-98dd-18c2fb415f0e.png)

focused valid inline input

![image](https://user-images.githubusercontent.com/8265238/41927904-2fc97b3a-7939-11e8-8f27-e4fe189765e2.png)

Reply:
I just checked the design kit and it looks like we are missing the background hover color, but there is also nothing defined for the error state for inline select. 
<img width="285" alt="screen shot 2018-06-26 at 12 41 02 pm" src="https://user-images.githubusercontent.com/2753488/41929597-36b22c6c-793e-11e8-95c9-5207faa3117a.png">

Reply:
sure I can go ahead and add the hover state in. The design kit may need to be updated to include data validation states in inline `<select>`s though because of #919

Reply:
@alisonjoseph actually regarding the hover state: Is it supposed to be part of the component's behavior? I think I would argue that this enhancement probably isn't necessary. Also right now there is a `--light` class modifier which will apply the white background in the default state. If we want to apply a background color change on hover, then there should probably be a spec for hovering over the light themed component too

Reply:
I'll defer that question to our designers @carbon-design-system/design Currently the spec only has the hover on the inline select without the --light modifier.

Reply:
any updates on this?

Reply:
So the inline select shouldn't have a background (or be set to transparent), except when hovered. Its supposed to be smaller with no background for smaller/more compact areas where a select is needed. The light modifier should not affect this component. A hover is necessary to indicate that the item is clickable to those using a mouse.

Reply:
thanks for the clarification, I have added the hover state changes to inline Select

Reply:
Thanks for taking care of this @emyarod! Reviewing now